content/docs/user-guide/how-to/run-dvc-on-windows.md

```diff
@@ -40,6 +40,50 @@ alternatives:
   [virtual machine](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/),
   or in an HD partition (dual boot).
 
+## Line endings
+
+If you are using Windows and you are working with people or environments (e.g.
+production) that are not, you’ll probably run into line-ending issues at some
+point. This is because Windows uses both a carriage-return character and a
+linefeed character for newlines in its files, whereas macOS and Linux systems
+use only the linefeed character.
+
+Since DVC is using content-based checksums for your pipeline dependencies,
+depending on your GIT configuration (see
+[core.autocrlf and core.eol config options](https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration)),
+DVC might see GIT-tracked files as changed, thus triggering pipeline
```

Comment:
```suggestion
depending on your Git configuration (see
[core.autocrlf and core.eol config options](https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration)),
DVC might see Git-tracked files as changed, thus triggering pipeline
```

Reply:
I'm constantly confused by the convention. I thought it should be the same as DVC